docs/getting-started/tutorial-full-experience.asciidoc

```diff
@@ -34,6 +34,13 @@ curl -O https://download.elastic.co/demos/kibana/gettingstarted/8.x/shakespeare.
 curl -O https://download.elastic.co/demos/kibana/gettingstarted/8.x/accounts.zip
 curl -O https://download.elastic.co/demos/kibana/gettingstarted/8.x/logs.jsonl.gz
 
+Alternatively, for Windows users, run the following commands in Powershell:
```

Comment:
For this line, could we tighten it up a bit? Something along the lines of:

If you are using Windows, run the following with Powershell: